File: PM_Backend/Common/src/main/java/common/dto/MasterUserDTO.java
package common.dto;
import com.fasterxml.jackson.annotation.JsonTypeInfo;

@JsonTypeInfo(
        use = JsonTypeInfo.Id.CLASS,
        include = JsonTypeInfo.As.PROPERTY,
        property = "@class"
)
public class MasterUserDTO extends DTO {
    private int id;
    private String masterUsername;
    private String masterPassword;

    public MasterUserDTO() {
    }

    public MasterUserDTO(String masterUsername, String masterPassword) {
        this.masterUsername = masterUsername;
        this.masterPassword = masterPassword;
    }

    public String getMasterUsername() {
        return masterUsername;
    }

    public void setMasterUsername(String masterUsername) {
        this.masterUsername = masterUsername;
    }

    public String getMasterPassword() {
        return masterPassword;
    }

    public void setMasterPassword(String masterPassword) {
        this.masterPassword = masterPassword;
    }

    @Override
    public int getId() {
        return id;
    }

    @Override
    public void setId(int id) {
        this.id = id;
    }
}

File: PM_Frontend/Shared/Dtos/MasterUserDTO.cs
using Newtonsoft.Json;

namespace Shared.Dtos
{
    public class MasterUserDTO : DTO
    {
        public string? masterUsername { get; set; }
        public string? masterPassword { get; set; }

        public MasterUserDTO(int? id, string? masterUsername, string? masterPassword) : base(id)
        {
            this.masterUsername = masterUsername;
            this.masterPassword = masterPassword;
        }

        public MasterUserDTO() { }

        public override string ToString()
        {
            string toString = "[";
            toString += masterUsername;
            toString += " ; ";
            toString += masterPassword;
            toString += "]";
            return toString;
        }
    }
}